### ISSUE TITLE
using * will comment the line when it shouldn't

### ISSUE THREAD

**Issue by Madrain** [TonyGravagno/MVBASIC-plus-plus#2]
using * will comment the line when you are trying to do a multiply operation as:
item3 = item1 * item2 
will treat item2 as comment

**Comment by TonyGravagno** [comment]
You're right. I's doing pure character checking, not real parsing/lexing. A parser would detect * at the beginning of the line or only after semi-colon ";". I'll see if I can make that smarter with the built-in tools. If not this will need to wait for a real language plugin, which is one of the intended goals for this project.
